src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java

```diff
@@ -542,7 +542,9 @@ public void withProfvis(String userAction, final Command command)
         "Preparing Profiler",
         userAction, 
         new Dependency[] {
-           Dependency.embeddedPackage("profvis")
+           Dependency.embeddedPackage("profvis"),
+           Dependency.cranPackage("htmlwidgets", "0.6"),
```

Comment:
One thing to pay attention to when specifying relatively recent dependencies (htmlwidgets 0.6 only went to CRAN a week or so ago) is that CRAN only builds Mac binaries for the current release of R and Windows binaries for the current and previous releases. This means that e.g. someone running R 3.0 (not current or previous) won't be able to install v0.6 as a binary. The workaround for this is typically to add the flag which indicates it's a source dependency (we can get burned by this if we ever add native code to a package but I don't think that will ever happen to htmlwidgets). I'll make the change to type = source, just wanted you to be aware of this issue for future reference.